README.md

```diff
@@ -393,6 +406,21 @@ directive.
 
 [Back to TOC](#table-of-contents)
 
+resty.kong.tag.get
+----------------------------------
+**syntax:** *resty.kong.tag.get()*
+
+**context:** *rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, log_by_lua&#42;*
```

Comment:
Is it usable in `header_filter` and `body_filter`?

Reply:
I think it can work in these phases, let me add some test cases.